packages/checkout-storefront/src/sections/PaymentSection/AdyenDropIn/createAdyenCheckout.ts

```diff
@@ -48,7 +52,6 @@ export function createAdyenCheckoutInstance(
     },
     onSubmit,
     onAdditionalDetails,
-    allowPaymentMethods: ["scheme"],
```

Comment:
Does this mean we don't have support for filtering only enabled payment methods?

Reply:
No, we don't.